sign_language_datasets/datasets/wmt_slt/wmt_slt.py

```diff
@@ -0,0 +1,242 @@
+        srf_poses_dir = path.join(srf_poses, 'srf', 'parallel')
+
+        srf_videos = self.download_zenodo_deposit(dl_manager, ZENODO_DEPOSIT_ID_SRF_VIDEOS_SUBTITLES,
+                                                  self.zenodo_srf_videos_token)
+        srf_parallel_dir = path.join(srf_videos, 'srf', 'parallel')
+
+        focusnews = self.download_zenodo_deposit(dl_manager, ZENODO_DEPOSIT_ID_FOCUS_NEWS, self.zenodo_focusnews_token)
+        focusnews_dir = path.join(focusnews, "focusnews")
+
+        datasets = {
+            'srf': {
+                'videos': path.join(srf_parallel_dir, 'videos'),  # "srf.2020-03-13.mp4"
+                'subtitles': path.join(srf_parallel_dir, 'subtitles'),  # "srf.2020-03-13.srt"
+                'openpose': path.join(srf_poses_dir, 'openpose'),  # "srf.2020-03-13.openpose.tar.xz"
+                'mediapipe': path.join(srf_poses_dir, 'mediapipe'),  # "srf.2020-03-13.mediapipe.tar.xz"
+            },
+            'focusnews': {
+                'videos': path.join(focusnews_dir, 'videos'),  # "focusnews.043.mp4"
+                'subtitles': path.join(focusnews_dir, 'subtitles'),  # "focusnews.043.srt"
+                'openpose': path.join(focusnews_dir, 'openpose'),  # "focusnews.043.openpose.tar.xz"
+                'mediapipe': path.join(focusnews_dir, 'mediapipe'),  # "focusnews.043.mediapipe.tar.xz"
+            }
+        }
+
+        print(f"SRF data stored in {srf_parallel_dir} and {srf_poses_dir}")
+        print(f"FocusNews data stored in {focusnews}")
+
+        return {
+            'train': self._generate_examples(datasets)
+        }
+
+    def _generate_examples(self, datasets):
+        """Yields examples."""
+
+        for dataset_id, directories in datasets.items():
+            names = [n[:-len('.mp4')] for n in os.listdir(directories['videos'])]
+            for name in names:
+                video_path = path.join(directories['videos'], f'{name}.mp4')
+                subtitles_path = path.join(directories['subtitles'], f'{name}.srt')
+
+                fps = get_framerate(video_path)
+
+                # # Load video
+                # video = None
+                # if self.builder_config.process_video:
+                #     video =
+
+                # Load poses
+                pose = None
+                if self.builder_config.include_pose is not None:
+                    if self.builder_config.include_pose == "openpose":
+                        openpose_path = path.join(directories['openpose'], f'{name}.openpose.tar.xz')
+                        pose = read_openpose_surrey_format(openpose_path, fps)
+
+                    if self.builder_config.include_pose == "holistic":
+                        mediapipe_path = path.join(directories['mediapipe'], f'{name}.mediapipe.tar.xz')
+                        pose = read_mediapipe_surrey_format(mediapipe_path, fps)
+
+                    reduce_pose_people(pose)
```

Comment:
Is there a reason why you reduce the number of people here? Instead of having the loader provide all poses?

Reply:
It is not possible to have a tensor with two unknown dimensions `(None, None, 135, 2)` :(